### PR TITLE
V0.0.3

### PR DESCRIPTION
**Pull request by bpavlov-daiteap**
Changes:
Redesign for success login page.
Add README file
Add user friendly output(in addition to json) for project list command
Changed package name to "daiteapcli"
Add global "output" flag
Changed "project" command to "projects"

### CODE REVIEW

**Comment by akafazov** [comment]
Missing Description for the PR.

File: cmd/daiteapcli/cmdUtils.go [hunk 1]
```diff
@@ -1,6 +1,8 @@
-package daiteap
+package daiteapcli
 
 import (
+	"fmt"
```
Comment (darashevcstbg) [inline cmd/daiteapcli/cmdUtils.go:4]: This is imported, but not used.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# Daiteapcli
+
+----
+
+Daiteapcli is a command line tool used for interaction with the Daiteap platform.
+
```
[the remark this answers is not in the document]
Comment (bpavlov-daiteap) [reply README.md:6]: Instructions are added

**Review by darashevcstbg — APPROVED** [review]

**Review by darashevcstbg — APPROVED** [review]
Looks OK.

**Review by akafazov — APPROVED** [review]

**Pull request merged**